### PR TITLE
7 : fix module syntax + npm publish lifecycle script

### PR DESCRIPTION
**Pull request by aktraore**

### CODE REVIEW

File: test/index.test.ts [hunk 1]
```diff
@@ -1,5 +1,6 @@
-import lunh from '../index';
 import { successfulStripeCreditCard, failedStripeCreditCard } from './stripe';
+import lunh = require('../index');
```
Comment (stvnyung) [inline test/index.test.ts]: Is there no actual other way to not have this ? It's so unconventinal I feel like we are giving up convention for ES6 module just to make node module works

**Comment by stvnyung** [comment]
#7

**Comment by stvnyung** [comment]
I used both `module.exports` (node module way) and `export default` (ES6 way) to have both working the conventional way.

I don't see any short coming but I would be interested in your guess here.

**Comment by aktraore** [comment]
as long as it works I don't see the problem, you'd have to test in node to see that it workds without issue

**Pull request merged**